web/styles/inbox.css

```diff
@@ -469,8 +471,8 @@
             }
 
             & i {
-                padding: 5px;
-                font-size: 16px;
+                padding: 0.3125em; /* 5px at 16px em */
+                font-size: 1em; /* 16px when font-size is 16px, scaled accordingly */
```

Comment:
I wonder if this is just a noop? Did you try removing it?

Reply:
I see that with a few other styles here, too. Probably worth taking a few seconds to switch them off in the browser inspector and seeing which are truly necessary.

Reply:
Ah yeah, I was able to remove them all, good call!